core/chains/evm/client/errors.go

```diff
@@ -424,6 +424,7 @@ func ClassifySendError(err error, lggr logger.Logger, tx *types.Transaction, fro
 		return commonclient.Fatal, err
 	}
 	if sendError.IsNonceTooLowError() || sendError.IsTransactionAlreadyMined() {
+		lggr.Debugw("Transaction already confirmed for this nonce: %d", tx.Nonce(), "err", sendError)
```

Comment:
For some of these messages we already log on the caller side, so you might want to aggregate those.

Reply:
Wouldn't it be alright to keep both? The caller would be able to provide more info potentially and it'd also make tracing the call easier. And keeping it the logs in the `ClassifySendError` ensures we get a log for all of these even if the caller side doesn't

Reply:
I've updated the error logging in the broadcaster and confirmer to avoid re-logging the same thing as the `ClassifySendError` function